### PR TITLE
Add parentheses surrounding cross-references

### PR DESCRIPTION
**Pull request by nedmsmith**
Updates section and table references that don't have surrounding parens.

See issue #286

### CODE REVIEW

**Review by deeglaze — APPROVED** [review]

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

File: draft-ietf-rats-corim.md [hunk 1]
```diff
@@ -286,29 +286,29 @@ constraints must be followed when creating or validating a CoRIM map.
...
-  Described in {{sec-common-validity}}
+  Described in ({{sec-common-validity}})
 
 * `entities` (index 5): A list of entities involved in a CoRIM life-cycle.
-  Described in {{sec-corim-entity}}
+  Described in ({{sec-corim-entity}})
 
 * `$$corim-map-extension`: This CDDL socket is used to add new information
-  structures to the `corim-map`.  See {{sec-iana-corim}}.
+  structures to the `corim-map`.  See ({{sec-iana-corim}}).
```
Comment (thomas-fossati) [inline draft-ietf-rats-corim.md:311]: These look spurious.  They'd render as:

> Described in (Section 1.2.3)

which is not what we want.  Instead, they should read as:

> Described in Section 1.2.3.

File: draft-ietf-rats-corim.md [hunk 2]
```diff
@@ -347,7 +347,7 @@ The following describes each child element of this type.
 * `href` (index 0): URI identifying the additional resource that can be fetched
 
 * `thumbprint` (index 1): expected digest of the resource referenced by `href`.
-  See {{sec-common-hash-entry}}.
+  See ({{sec-common-hash-entry}}).
```
Comment (thomas-fossati) [inline draft-ietf-rats-corim.md:350]: ditto

File: draft-ietf-rats-corim.md [hunk 3]
```diff
@@ -457,10 +457,10 @@ This ensures the consumer is able to identify credentials used to authenticate i
 The following describes each child item of this group.
 
 * `signer` (index 0): Information about the entity that signs the CoRIM.
-  Described in {{sec-corim-signer}}
+  Described in ({{sec-corim-signer}})
 
 * `signature-validity` (index 1): Validity period for the CoRIM. Described in
-  {{sec-common-validity}}
+  ({{sec-common-validity}})
```
Comment (thomas-fossati) [inline draft-ietf-rats-corim.md:463]: ditto

File: draft-ietf-rats-corim.md [hunk 4]
```diff
@@ -526,13 +526,13 @@ The following describes each member of the `concise-mid-tag` map.
...
+  identification information for the CoMID. Described in ({{sec-comid-tag-id}}).
 
 * `entities` (index 2): Provides information about one or more organizations
-  responsible for producing the CoMID tag. Described in {{sec-comid-entity}}.
+  responsible for producing the CoMID tag. Described in ({{sec-comid-entity}}).
 
-* `linked-tags` (index 3): A list of one or more `linked-tag-map` (described in
-  {{sec-comid-linked-tag}}), providing typed relationships between this and
+* `linked-tags` (index 3): A list of one or more `linked-tag-map`. Described in
+  ({{sec-comid-linked-tag}}), providing typed relationships between this and
```
Comment (thomas-fossati) [inline draft-ietf-rats-corim.md:535]: ditto, and below too.

**Comment by nedmsmith** [comment]
See if commit https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/300/commits/1372934797b8e6ca7018730564e1a430a74fe8f4 addresses your concerns.

**Comment by thomas-fossati** [comment]
> See if commit [1372934](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/commit/1372934797b8e6ca7018730564e1a430a74fe8f4) addresses your concerns.

Thanks for the changes!

**Review by thomas-fossati — APPROVED** [review]

**Pull request merged**